### ISSUE TITLE
Few errors when trying the examples

### ISSUE THREAD

**Issue by jamel-mes** [rmarkello/snfpy#10]
Hi,
I am trying to follow your examples but I get this error when I try to load datasets:
```
from snf import datasets
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-40-ba839aa43043> in <module>
----> 1 from snf import datasets

ImportError: cannot import name 'datasets'
```

- In the examples, I am not able to find snf.snf() , is it snf.SNF() ?

- In addition, could you please upload data1.csv & data2.csv to have an idea of the numpy array shape?

**Comment by rmarkello** [comment]
Hi @jamel-mes!

I'm so sorry about that&mdash;I've been actively updating the code in `snfpy` and some of the documentation had fallen out of sync with the package internals. I've just made a few more updates and pushed a new release, so if you could install the latest version of `snfpy` (e.g., with `pip install -U snfpy`) and then confirm you have the right version:

```python
>>> import snf
>>> snf.__version__
'0.2'
```

then all of the examples should work, again. Let me know if that seems to work for you, or if you have any more issues / questions!

**Comment by jamel-mes** [comment]
Thank you @rmarkello for the response! I updated the snf package using `pip install -U snfpy`
However I still get an error when trying to load the datasets:
```
from snf import datasets
digits = datasets.load_digits()
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-04cf7bc4a0b4> in <module>
      1 from snf import datasets
----> 2 digits = datasets.load_digits()

/apps/anaconda3/envs/snfpy/lib/python3.6/site-packages/snf/datasets.py in load_digits()
     79     ]
     80 
---> 81     return _load_data('digits', dfiles)

/apps/anaconda3/envs/snfpy/lib/python3.6/site-packages/snf/datasets.py in _load_data(dset, dfiles)
     36                          'or snf.datasets.load_digits() it is possible that '
     37                          'snfpy was improperly installed. Please check your '
---> 38                          'installation and try again.'.format(dset))
     39 
     40     # space versus comma-delimited files (ugh)

ValueError: digits is not a valid dataset. If you are receiving this error after using snf.datasets.load_simdata() or snf.datasets.load_digits() it is possible that snfpy was improperly installed. Please check your installation and try again.
```

**Comment by rmarkello** [comment]
Oof! Thanks for pointing out this glaring error :grimacing:  I think I just resolved the issue with #12 and pushed a new release (version 0.2.1) to PyPi, so if you could rerun `pip install -U snfpy` and try _one more time_ I think that things should work.

Let me know, and thank you again for the bug report! :raised_hands:

**Comment by jamel-mes** [comment]
works now, thank you!
